src/main/java/io/supertokens/storage/mysql/Start.java

```diff
@@ -1112,16 +1138,11 @@ public void createCode(PasswordlessCode code) throws StorageQueryException, Unkn
         try {
             PasswordlessQueries.createCode(this, code);
         } catch (StorageTransactionLogicException e) {
-            // TODO: review these exceptions
             String message = e.actualException.getMessage();
 
             if (e.actualException instanceof UnknownDeviceIdHash) {
                 throw (UnknownDeviceIdHash) e.actualException;
             }
```

Comment:
PasswordlessQueries should not throw `UnknownDeviceIdHash`. The message check should still happen in this file